Added Declare lost, tweaked data

sxqnicholas96 opened the pull request.

auskure commented:
hey could i ask why the minor changes for the data is needed?  :O @sxqnicholas96

sxqnicholas96 commented:
@auskure It's so i can demonstrate your trigger checks lolol. What i did was i added a invoicedLoan in the future and I removed the update isReturned true for Emerson Franks for her latest invoicedLoan she had.

auskure commented:
haha okay fair enough :O
